pennylane/_qubit_device.py

```diff
@@ -203,16 +205,18 @@ def execute(self, circuit, **kwargs):
                 if shot_tuple.copies > 1:
                     results.extend(r.T)
                 else:
-                    results.append(r)
+                    results.append(r.T)
```

Comment:
When having multiple `qml.sample` expressions in the return statement, this was required such that the resulting arrays  have the same shape for both when `copies=1` and `copies>1`. (Without this, there's a mismatch, e.g., `shots=[10]` produced `(2,10)` and `shots=[10,10,10]` produced `(10,2)` 3 times).

Reply:
It had no effect on the existing tests

Reply:
As for the demos, [attempted to run the CI](https://app.circleci.com/pipelines/github/PennyLaneAI/qml/2833/workflows/d7e6404a-68a2-4ffe-8d55-62827c7cff62/jobs/2992), though ran into a requirements conflict due to the mismatch in the branches (we require the branch to be `master`).

Reply:
Thanks @antalszava!